Controller/PageController.php

```diff
@@ -107,7 +107,7 @@ protected function getCmsManagerSelector()
     /**
      * @return Request
      */
-    private function getRequest()
+    private function getRequestObject()
```

Comment:
I think the method itself could be greatly simplified. `request_stack` is guaranteed to exist now, isn't it?

Reply:
I  did this cleanup already in [PR #886](https://github.com/sonata-project/SonataPageBundle/pull/886/files#diff-004d0d5de4293bbc0343af54d205429eL112). But I can move that refactor to this PR if you like!

Reply:
Oh no, it's perfect, don't bother!